home/views.py

```diff
@@ -1,10 +1,17 @@
 from django.shortcuts import render
 
+from hackathon.models import Hackathon
+
 
 def index(request):
     """ A view to return the index page """
+    hackathons = Hackathon.objects.all()
+
+    context = {
```

Comment:
I don't think you need to declare an extra `context` dict for only one key, just pass the dict directly into the `render` function.

Reply:
When doing that I get the error, "context must be a dict and not a queryset". I also figured that we might want to declare extra information for the landing page in future as the project grows.